app/importers/mahonia_validator.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class MahoniaValidator < Darlingtonia::Validator
```

Comment:
Could we just drop this validator, it's unused in production right? Do you have a future use in mind?